src/library/Uno.Themes.WinUI.Markup/Theme/HyperlinkButton.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class HyperlinkButton
+		{
+			public static class Resources
+			{
+				public static class Foreground
+				{
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonForeground")]
+					public static ResourceValue<Brush> Default => new("HyperlinkButtonForeground", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonForegroundPointerOver")]
+					public static ResourceValue<Brush> PointerOver => new("HyperlinkButtonForegroundPointerOver", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonForegroundPressed")]
+					public static ResourceValue<Brush> Pressed => new("HyperlinkButtonForegroundPressed", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "HyperlinkButtonForegroundDisabled")]
+					public static ResourceValue<Brush> Disabled => new("HyperlinkButtonForegroundDisabled", true);
+
+					[ResourceKeyDefinition(typeof(Brush), "SecondaryHyperlinkButtonForeground")]
+					public static ResourceValue<Brush> Secondary => new("SecondaryHyperlinkButtonForeground", true);
```

Comment:
This resource was removed [here](https://github.com/unoplatform/Uno.Themes/pull/1092/files) so we can remove it here